css/style.css

```
@@ -198,4 +206,155 @@ footer {
   color: #FFFFFF;
   text-decoration: none;
   margin: 0 10px;
+}
+
+.sns {
+  display: flex;
+  gap: 10px;
+}
+
+
+/* Media Query */
+
+/* Tablet: 768px 이상 ~ 1199px 이하 */
+@media (max-width: 1199px) {
+  .banner {
+    padding: 0 24px;
+  }
+
+  .top-section,
+  .bottom-section {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 480px;
+  }
+
+  .top-content,
+  .bottom-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-end;
+    text-align: center;
+  }
+
+  .top-title h2,
+  .bottom-title h2 {
+    font-size: 27px;
+    margin: 0 auto;
+    width: 100%;
+    line-height: 1.5;
+    margin-bottom: 10px;
+  }
+
+  /* 적용이 안되는 이유를 못찾음 */
+  .top-title h2.no-wrap {
+    white-space: nowrap;
+  }
+
+  .top-title a {
+    margin: 0 auto;
+  }
+
+  .top-content img,
+  .bottom-content img {
+    align-items: center;
+    align-self: center;
+    margin-top: 70px;
+    width: 520px;
+  }
+
+  .middle-section {
+    flex-grow: 1;
+  }
+
+  .hot-item,
+  .search,
+  .register {
+    flex-direction: column;
+  }
+
+  .hot-item img,
+  .search img,
+  .register img {
+    width: 600px;
+    height: 400px;

```

❗️ 수정요청
이렇게 height, width 속성을 동시에 주게 되시면 이미지의 원본 비율을 고려해 작성해주신 것이 아니라면 지금처럼 비율이 달라지게 됩니다!
구현할때 이미지의 비율을 유지하는 것이 중요하므로 비율을 유지할 수 있는 방향으로 수정해보세요!

```suggestion
    width: 600px;
    height: auto; // 이미지의 원본 비율을 유지하면서 자동으로 늘어나게 설정
```